README.md

```diff
@@ -2,30 +2,68 @@
+   - Each week a release team will check tracked VEP in order to assure activity
 
+5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement includes all the relevant information, including the design and the state.
+
+The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated). 
+
+The resposiblity of Approver is to make sure the VEP is implemented, not divering and implementation is not lacking behind VEP, following is nonexhaustive list meant as checklist:
+1. After Feature Freeze, approvers needs to go over issue and perform the following checklist
    1. All PRs are merged into release branch
    2. Docs PR is merged (plan review ahead of release if only placeholder is opened)
-   3. Verify that Enhancement was implemented and doesn't need any update
+   3. Verify that Enhancement was implemented and doesn't need any update or exception
    4. Track any bugs
+   5. Make sure the VEP's issue is tracking required PRs and Issues
 2. Weekly check-in on progress of Enhancement and its implementation
 3. Coordinate sigs, reviewers and approvers in order to progress the Enhancement
 
+**Release check-ins**
+Both the release team and approvers of the VEPs are responsible for weekly check-ins. The following are the goals of the check-ins:
+1. Re-targeting of VEP - In case of implementation not converging, new blockers being discovered, pushback of community or withdrawal of approver the VEP needs to be re-targeted. The VEP needs to be updated with new target. The SIGs should shift the focus on tracked VEPs. 
+Re-targeting could be also rejection of the VEP completly in case it is not implentable.
+
+2. Coordination - SIGs are responsible that the reviews are not lacking behind for more than week. Release team is making sure there is an active SIG representative.
+
+**Labels**
+For easier managment of the release and VEPs the following labels will be used:
+
+1. SIG labels - Each SIG will have a label in order to sort what SIGs is responsible for the VEP.
+2. Target labels - There will be a lable in order to target the VEP for release.
+
+**NOTE**
+Acceptance of enhancement doesn't guarantee that the feature will land in the current or later release. The process is collaborative effort between contributor/s and Approvers. Features not landing to release branch prior to FF will need to file for excepion, see <TODO>
+
+## Deadlines
+The particular dealines are always changing based on the release and are publish <Link to repo>. The following deadlines are important for the VEP:
+1. Enhancement Freeze - The deadline for this milestone is Alpha release of KubeVirt. See <TODO link to release repo>
+2. Feature Freeze - This is tracked by each release <TODO link to release repo>
```

Comment:
Maybe add another step before these two:
* VEP planning - at the beginning of every release cycle, each SIG would prioritize VEPs and decide which ones are being tracked for the upcoming release.